src/app/frontend/zerostate/zerostate.scss

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+$primary-color: #326DE6;
```

Comment:
Can you apply a class from the angular-material theme, so that you do not need to redefine the color? See documentation at: https://material.angularjs.org/latest/Theming/01_introduction or their github.

For example, how about applying md-toolbar class to this?